python_modules/dagster-graphql/dagster_graphql_tests/graphql/conftest.py

```diff
@@ -19,7 +19,11 @@ def graphql_context():
                     "module": "dagster.utils.test",
                     "class": "FilesystemTestScheduler",
                     "config": {"base_dir": temp_dir},
-                }
+                },
+                "run_coordinator": {
+                    "module": "dagster._core.run_coordinator.immediately_launch_run_coordinator",
+                    "class": "ImmediatelyLaunchRunCoordinator",
+                },
```

Comment:
do any of our test suites use now default run coordinator? since these are often effectively used as e2e integration tests it would be good if at least one of the test cases matched the actual default behavior